### ISSUE TITLE
Expose hotkeys to Lua

### ISSUE THREAD

**Issue by Mailaender** [OpenRA/OpenRA#20222]
My idea is to allow tutorial style messages like `Press $HOTKEY to force fire`.

**Comment by anvilvapre** [comment]
To receive notifications of hotkey (press) events? Or to also be able to assign them?

In case of the last - better not allow override of existing hotkeys. As it would probable get abused.

**Comment by Mailaender** [comment]
No, not assign. Just show `Press CTRL + A to force move` and yes, maybe also check if it got pressed for interactive tutorials, so they can continue.
